unity-renderer/Assets/DCLPlugins/EquippedEmotes/EquippedEmotesPlugin.cs

```diff
@@ -0,0 +1,106 @@
+using DCL.Helpers;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace DCL.EquippedEmotes
+{
+    /// <summary>
+    /// Plugin feature that initialize the Equipped Emotes feature.
+    /// </summary>
+    public class EquippedEmotesPlugin : IPlugin
+    {
+        internal const string PLAYER_PREFS_EQUIPPED_EMOTES_KEY = "EquippedNFTEmotes";
+
+        internal DataStore_EmotesCustomization emotesCustomizationDataStore => DataStore.i.emotesCustomization;
+
+        public EquippedEmotesPlugin()
+        {
+            LoadDefaultEquippedEmotes();
+
+            emotesCustomizationDataStore.isEmotesCustomizationInitialized.OnChange += OnEmotesCustomizationInitialized;
+            OnEmotesCustomizationInitialized(emotesCustomizationDataStore.isEmotesCustomizationInitialized.Get(), false);
+
+            emotesCustomizationDataStore.equippedEmotes.OnSet += SaveEquippedEmotesInLocalStorage;
```

Comment:
Maybe we should also subscribe to `.Add`  and `.Remove` for future proof changes.

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2078/commits/01c8aff3a2548f8f13f86d909574acb894cad1c8.